assets/javascripts/discourse/templates/connectors/flagged-post-below-controls/sift-flag-queue.hbs

```diff
@@ -3,10 +3,13 @@
 
     <div>
     {{#each-in flaggedPost.custom_fields.sift.topics as |topicKey topicValue|}}
+        <!-- following code line is a custom 'if' condition check via a helper to evaluate if the topicValue -->
+        {{#if (ifCond topicValue '>' 4)}}
```

Comment:
I think we want to include 4s in the topic badges, as we are showing them in the text classification